source/grid/manifold_lib.cc

```diff
@@ -462,6 +508,8 @@ SphericalManifold<dim, spacedim>::normal_vector(
     // If it is not a horizontal face, just use the machinery of the
     // base class.
     return Manifold<dim, spacedim>::normal_vector(face, p);
+
+  return Tensor<1,spacedim>();
```

Comment:
Why would this be necessary? You terminate both the `if` and `else` branch above in a `return` statement...

Reply:
I think this was to fix a warning

Reply:
Yes, without this I get a warning about the possibility of never reaching a return statement in gcc 9.2.1. Not sure why, `if-else` should always reach a return, but I guess gcc is worried that one branch may contain a `break` statement before the `return`. Anyway, this is something that came up in a few places in ASPECT already (and likely in deal.II in other places too) and our usual solution was to just add a return that will never trigger.